app/test_split_integer.py

```diff
@@ -2,20 +2,46 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 20
+    number_of_parts = 3
+
+    assert sum(split_integer(value, number_of_parts)) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 21
+    number_of_parts = 3
+
+    result = split_integer(value, number_of_parts)
+
+    if value % number_of_parts:
+        assert (max(result) - min(result)) == 1
+    else:
+        assert (max(result) - min(result)) == 0
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    value = 20
+    number_of_parts = 1
+
+    result = split_integer(value, number_of_parts)
+
+    assert result[0] == value
```

Comment:
when writing tests, it's  clearer to compare the expected output directly rather than individual elements of the output.

compare the result with the expected output list